CONTRIBUTING.md

```diff
@@ -98,8 +98,8 @@ Not sure what to write about, take a look at our [issues](https://github.com/sel
 
 ## Other Resources
 
-For help with Markdown:
-
-[Markdown -- Github Help](https://help.github.com/en/github/writing-on-github)
-
-[Markdown Cheatsheet](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)
+For help with Markdown:  
+[MkDocs Cheatsheet ](https://3os.org/markdownCheatSheet/welcome/)  
+[MkDocs -- Admonition](https://3os.org/markdownCheatSheet/admonition/)  
```

Comment:
Probably worth linking to the `mkdocs-material` docs here instead https://squidfunk.github.io/mkdocs-material/reference/admonitions/

Reply:
I was not aware of this link. Nice.